sound/soc/codecs/max98373.c

```diff
@@ -780,6 +780,13 @@ static int max98373_probe(struct snd_soc_component *component)
 	regmap_write(max98373->regmap,
 		MAX98373_R202A_PCM_TO_SPK_MONO_MIX_2,
 		0x1);
+	/* Set initial volume (0dB) */
+	regmap_write(max98373->regmap,
+		     MAX98373_R203D_AMP_DIG_VOL_CTRL,
+		     0x00);
+	regmap_write(max98373->regmap,
+		     MAX98373_R203E_AMP_PATH_GAIN,
+		     0x00);
```

Comment:
this doesn't look good, we removed this on purpose to take Mark Brown's into account, not sure why this needed to be added again?